plugins/slack_webhooks/app/models/slack_webhook_notification.rb

```diff
@@ -17,7 +17,8 @@ def buddy_request(message)
 
   def default_buddy_request_message
     project = @deploy.project
-    ":pray: @here _#{@deploy.user.name}_ is requesting approval" \
+    # https://api.slack.com/docs/message-formatting
+    ":pray: <!here> _#{@deploy.user.name}_ is requesting approval" \
```

Comment:
I attempted to format the the deploy url nicely with proper slack message formatting but failed due to the same issue. I could give it a second try. :)